v4_x/git-remote-gosh/src/git_helper/push/parallel_snapshot_upload_support.rs

```diff
@@ -330,10 +331,17 @@ impl ParallelTreeUploadSupport {
         let remote_network = context.remote.network.clone();
         let repo = context.remote.repo.clone();
 
-        tracing::trace!("Start push of tree: address: {tree_address:?}");
-
-        self.expecting_deployed_contacts_addresses
-            .push(tree_address.clone());
+        let tree_contract =
```

Comment:
Bad logic here: 
If tree was deployed, we still need to check that it is ready and push it's address to expected